mteb/tasks/Classification/multilingual/TurkicClassification.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from collections import Counter
+
+import datasets
+from datasets import DatasetDict
+
+from mteb.abstasks import AbsTaskClassification, MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class TurkicClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TurkicClassification",
+        description="A dataset of news classification in three Turkic languages.",
+        dataset={
+            "path": "Electrotubbie/classification_Turkic_languages",
+            "revision": "db1a67c1bdd54fbb8536af026dc8596f00f9c41d",
+        },
+        reference="https://huggingface.co/datasets/Electrotubbie/classification_Turkic_languages/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs={
+            "ky": ["kir-Cyrl"],
+            "kk": ["kaz-Cyrl"],
+            "ba": ["bak-Cyrl"],
+        },
+        main_score="accuracy",
+        date=("2023-02-16", "2023-09-03"),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license="Not specified",
+        socioeconomic_status="low",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        """,
+        n_samples={"train": 193056},
+        avg_character_length={"train": 1103.13},
+    )
+
+    def transform_data(self, dataset, lang):
+        dataset_lang = DatasetDict()
+        label_count = Counter(dataset["train"]["label"])
+        dataset_lang["train"] = dataset["train"].filter(
+            lambda example: example["lang"] == lang
+            and label_count[example["label"]] >= 20
+        )
```

Comment:
here you just need to filter examples of that lang right? you're anyway doing `stratified_subsampling` on label after this

Reply:
There were too many labels with ver few occurrences, so I made a choice to prune out labels that have less than 20 occurrences.